index.js

```diff
@@ -1,28 +1,65 @@
 'use strict';
+const fs = require('fs');
 const semver = require('semver');
 const execa = require('execa');
 const del = require('del');
-const chalk = require('chalk');
-
+const pify = require('pify');
+const Listr = require('listr');
+const split = require('split');
+require('any-observable/register/rxjs-all');
+const Observable = require('any-observable');
+const streamToObservable = require('stream-to-observable');
+
+const fsP = pify(fs);
 const VERSIONS = ['major', 'minor', 'patch', 'premajor', 'preminor', 'prepatch', 'prerelease'];
 
-const log = str => console.log(chalk.bold(`\n${chalk.cyan('›')} ${str}\n`));
-
 const exec = (cmd, args) => {
-	// TODO Switch to `{stdio: 'inherit'}` instead of manual logging when a new execa version is released
-	const result = execa.sync(cmd, args);
+	// Use `Observable` support if merged https://github.com/sindresorhus/execa/pull/26
+	const cp = execa(cmd, args);
 
-	if (result.stdout) {
-		console.log(result.stdout);
-	}
+	return Observable.merge(
+		streamToObservable(cp.stdout.pipe(split()), {await: cp}),
+		streamToObservable(cp.stderr.pipe(split()), {await: cp})
+	).filter(Boolean);
```

Comment:
Not sure if necessary but should we add trimming here to make sure the offset is gone. Did not saw an example of that but it seems quite obvious that it's possible.

```js
return Observable.merge(
	streamToObservable(cp.stdout.pipe(split()), {await: cp}),
	streamToObservable(cp.stderr.pipe(split()), {await: cp})
).map(x => x.trim()).filter(Boolean);
```